seed/checkUserExistence.js

```diff
@@ -5,11 +5,13 @@ const {
 } = require('~/consts/auth')
 const SeedSuperAdmin = require('~/seed/seedSuperAdmin')
 const checkRoleExistence = require('~/seed/checkRoleExistence')
+const checkCategoryExistence = require('~/seed/checkCategoryExistence')
 const logger = require('~/logger/logger')
 
 const checkUserExistence = async () => {
   try {
     await checkRoleExistence()
+    await checkCategoryExistence()
```

Comment:
i am pretty sure you can move it to serverSetup.js file, user creation doesn't depend on category creation

Reply:
moved.